codespell_lib/data/dictionary.txt

```diff
@@ -27729,14 +27869,26 @@ hopyfully->hopefully
 horicontal->horizontal
 horicontally->horizontally
 horinzontal->horizontal
+horison->horizon
+horisons->horizons
+horisontal->horizontal
+horisontally->horizontally
 horizntal->horizontal
 horizonal->horizontal
 horizonally->horizontally
 horizontale->horizontal
+horizontall->horizontal, horizontally,
+horizontallly->horizontally
+horizontaly->horizontally
 horiztonal->horizontal
 horiztonally->horizontally
+horozon->horizon
+horozons->horizons
+horozontal->horizontal
+horozontally->horizontally
 horphan->orphan
 horrable->horrible
+horray->hooray
```

Comment:
Can also be spelled `hurray`, but `hooray` is definitely more common:
https://books.google.com/ngrams/graph?content=hooray%2Churray&year_start=1800&year_end=2019&corpus=en-2019

Let's suggest `hooray` only.